next/components/Consortia/CSER/components/Projects/common/entities.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention -- TODO update eslint rule.
```

Comment:
@NoopDog, I believe for AnVIL next we need the eslint rule to accept uppercase enum?